.github/workflows/keepalive.yml

```diff
@@ -17,7 +17,19 @@ jobs:
       - uses: actions/checkout@v4
         with:
           persist-credentials: 'true'
+      - name: Merge the upstream default branch
+        id: merge
+        if: github.event.repository.fork == true
+        run: |
+          git fetch https://github.com/gitgitgadget/gitgitgadget-workflows HEAD &&
```

Comment:
Can this information be obtained from context info like github.event.repository?  Just looking to make it generic.

Reply:
That is unfortunate.  Maybe `gitgitgadget/gitgitgadget-workflows` could be in a variable to help make it clear to anyone using this model that this is must be changed.

Not something to hold back this PR.